[Zaraz] Clarify Zaraz procedures (#3732)
cloudflare/cloudflare-docs: content/zaraz/web-api/zaraz-set.md

# Zaraz Set

> Sometimes you might want to make a variable available in all your events without manually setting it every time you are using `zaraz.track()`. For the purposes of this example, we are going to assume users in your system have a unique identifier that you want to send to your tools. You might have many `zaraz.track()` calls all sharing this one parameter:

Reviewer: ```suggestion
You may want to make a variable available in all your events without manually setting it every time you are using `zaraz.track()`. For the purpose of this example, assume users in your system have a unique identifier that you want to send to your tools. You might have many `zaraz.track()` calls all sharing this one parameter:
```

```js
zaraz.track("form completed", {userId: "ABC-123"})
```

```js
zaraz.track("button clicked", {userId: “ABC-123”, value: 200})
```

```js
zaraz.track("cart viewed", {items: 3, userId: “ABC-123”})
```

> As you can see, all the events are collecting the `userId` key, and the code for setting that key repeats itself. With `zaraz.set(key, value, [options])` you can avoid repetition by setting the key once when the page loads. Zaraz will then attach this key to all future `zaraz.track()` calls. In the above example, you can remove the `userId` key from all `zaraz.track()` calls, if you call `zaraz.set(“userId”, “ABC-123”)` once, before the `zaraz.track()` calls.

Reviewer: ```suggestion
In the provided example, all the events are collecting the `userId` key, and the code for setting that key repeats itself. With `zaraz.set(key, value, [options])` you can avoid repetition by setting the key once when the page loads. Zaraz will then attach this key to all future `zaraz.track()` calls. In the above example, you can remove the `userId` key from all `zaraz.track()` calls, if you call `zaraz.set(“userId”, “ABC-123”)` once, before the `zaraz.track()` calls.
```
Removing 'see' language for a potential substitute

Author: Great catch!
